projects/plugins/jetpack/_inc/client/components/apps-card/style.scss

```diff
@@ -26,22 +26,21 @@
 }
 
 .jp-apps-card__top {
-	padding: rem( 60px ) 0 0;
-	background: #ffffff;
+	padding: rem( 25px ) 0 0;
+	background: #fff;
 	text-align: center;
 
 	img {
 		max-width: 26%;
-		padding-top: 10px;
 	}
 }
 
 .jp-apps-card__description {
 	max-width: 80%;
 	margin: 0 auto;
-	padding: rem( 10px ) rem( 24px ) rem( 24px );
+	padding: rem( 10px ) rem( 30px ) rem( 25px );
```

Comment:
The `max-width` in combination with the `padding` creates a fairly narrow space for the text on small devices. WDYT about removing the `max-width` and reducing the horizontal `padding`? If we are concerned about illegible line length, we could apply an `em`-based `max-width` to `.jp-apps-card__header`/`.jp-apps-card__paragraph` instead. 

<img width="322" alt="Narrow small screen content width" src="https://user-images.githubusercontent.com/438664/192859059-a7d491a4-758a-4083-ac57-c85bbb649f5c.png">

Reply:
@dcalhoun, thank you! I implemented those suggested changes in https://github.com/Automattic/jetpack/pull/26276/commits/52bcdf75ac819c1fd258e0cf2c449eed82748d18. I didn't see a need to set a max-width to `max-width` to `.jp-apps-card__header`/`.jp-apps-card__paragraph` as, even with removing the `max-width` from the container and reducing the horizontal padding, the padding seems enough to prevent illegible text.

Tested with a Galaxy Fold emulator _(280 x 683)_:

| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/193103345-126cd768-8c51-4f7e-aad4-c41127107a22.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/193102590-65176fe5-17a6-4c62-b93c-f602bc606098.png" width="100%"> |

Let me know if you think this needs further tweaking!

Reply:
Looks good! 🙇🏻